### DISCUSSION TITLE
Restarting After Stopping?

### DISCUSSION THREAD

**Discussion by bd8993** [Hexagon/croner#216] [Q&A]
It doesn't seem like there are any plans or options to restart a cron after you have decided to stop it. I see there is a pause and a resume method so I just wanted to clarify and understand what the purpose of stop is. In my head, pause should not allow the cron to run on any future times that it's cron pattern would match unless it gets resumed. So with pause stopping future action, I feel like stop should kill a currently running cron function, but it seems to also pause all future cron runs. That's not a bad thing necessarily, but it seems odd that there is then no way to ever resume the stopped job other than restarting the entire script. Is there any reason to never allow a stopped cron to be ran again? I know you can manually trigger it later, but I guess im just confused on the use-case of stop

**Comment by Hexagon** [comment]
The use case of stop is to prevent any further runs - stopped is stopped forever. Then there is pause/resume as you've seen. 

The use case for stop is to make sure a job can never run again, to make sure it's safe to deleted it (if there are references kept).
